pkg/plugins/policies/meshretry/api/v1alpha1/schema.yaml

```diff
@@ -0,0 +1,171 @@
+                    backOff:
+                      description: BackOff is a configuration of durations which will be used in exponential backoff strategy between retries
+                      properties:
+                        baseInterval:
+                          description: BaseInterval is an amount of time which should be taken between retries. Must be greater than zero. Values less than 1 ms are rounded up to 1 ms. Default is 25ms.
+                          type: string
+                        maxInterval:
+                          description: MaxInterval is a maximal amount of time which will be taken between retries. Default is 10 times the "BaseInterval".
+                          type: string
+                      type: object
+                    numRetries:
+                      description: NumRetries is the number of attempts that will be made on failed (and retriable) requests
+                      format: int32
+                      type: integer
+                    perTryTimeout:
+                      description: PerTryTimeout is the amount of time after which retry attempt should timeout. Setting this timeout to 0 will disable it. Default is 15s.
+                      type: string
+                    retriableRequestHeaders:
+                      description: RetriableRequestHeaders is an HTTP headers which must be present in the request for retries to be attempted.
+                      items:
+                        properties:
+                          name:
+                            type: string
+                          type:
+                            enum:
+                              - REGULAR_EXPRESSION
+                              - EXACT
+                              - PREFIX
+                            type: string
+                          value:
+                            type: string
+                        type: object
+                      nullable: true
+                      type: array
+                    retriableResponseHeaders:
+                      description: RetriableResponseHeaders is an HTTP response headers that trigger a retry if present in the response. A retry will be triggered if any of the header matches match the upstream response headers.
+                      items:
+                        properties:
+                          name:
+                            type: string
+                          type:
+                            enum:
+                              - REGULAR_EXPRESSION
+                              - EXACT
+                              - PREFIX
+                            type: string
+                          value:
+                            type: string
+                        type: object
+                      nullable: true
+                      type: array
+                    retryOn:
+                      description: RetryOn is a list of conditions which will cause a retry.
```

Comment:
same here

Reply:
Yes, we can do this for GRPC. I wanted this to be the same type as `http.retryOn`, but we can convert `grpc.retryOn` to enum

Reply:
I'm ok with this staying as string if we specify all values that we can put here

Reply:
Updated the description